source/agora/registry/main.d

```diff
@@ -84,16 +105,17 @@ private int main (string[] strargs)
     so client connections should not lead to `Exception` escaping this function.
 
     Params:
+      config = Registry configuration
       registry = The name registry to forward the queries to.
 
 *******************************************************************************/
 
-private void runDNSServer_canThrow (NameRegistry registry)
+private void runDNSServer_canThrow (in Config config, NameRegistry registry)
 {
     // The `listenUDP` needs to be in the `runTask` otherwise we get
     // a fatal error due to a bug in vibe-core (see comment #2):
     /// https://github.com/vibe-d/vibe-core/issues/289
-    auto udp = listenUDP(53);
+    auto udp = listenUDP(config.dns.port, config.dns.address);
```

Comment:
why did this become two parameters from one?

Reply:
Because the default value is `"0.0.0.0"`, and now I've made it configurable.
https://vibed.org/api/vibe.core.net/listenUDP